packages/backend/docker-compose.yml

```diff
@@ -2,11 +2,11 @@ version: "3.9"
 # Creates local network of peers
 
 x-with-tor: &with-tor
-    BOOTSTRAP_ADDRS: /dns4/ix2oumqrtjaupt53l6cqpk6ct6iaa5guconwgtvgdk2v3i5wjiyehryd.onion/tcp/7788/ws/p2p/QmRbkBkhTt2DbLMF8kAaf1oxpfKQuEfLKFzVCDzQhabwkw
+    BOOTSTRAP_ADDRS: /dns4/ix2oumqrtjaupt53l6cqpk6ct6iaa5guconwgtvgdk2v3i5wjiyehryd.onion/tcp/7788/ws/p2p/12D3KooWPYjyHnYYwe3kzEESMVbpAUHkQyEQpRHehH8QYtGRntVn
```

Comment:
Any of these changes to peer IDs that are hardcoded are just for testing and because of the change in peer ID format.